roles/foreman_provisioning/tasks/configure_centos_9.yml

```diff
@@ -0,0 +1,27 @@
+- name: "Ensure CentOS Stream 9"
+  theforeman.foreman.operatingsystem:
+    username: "{{ foreman_provisioning_username }}"
+    password: "{{ foreman_provisioning_password }}"
+    server_url: "{{ foreman_provisioning_server_url }}"
+    name: CentOS Stream
```

Comment:
Names can't contain spaces. So it needs to be "CentOS" or "CentOSStream" or "CentOS_Stream".

Looking at Foreman core, we use `CentOS_Stream` there, so I'd keep that.